src/app-layout/classic.tsx

```diff
@@ -0,0 +1,634 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import clsx from 'clsx';
```

Comment:
Git did not manage to follow the files rename,  this code moved from `index.tsx` without changes